src/devTools.js

```diff
@@ -95,13 +110,26 @@ class DevToolsEnhancer {
     }
   }
 
+  importPayloadFrom = (store, state, instance) => {
+    try {
+      const nextLiftedState = importState(state, instance);
+      if (!nextLiftedState) return;
+      store.liftedStore.dispatch({ type: 'IMPORT_STATE', ...nextLiftedState });
+      this.relay(
+        'STATE',
+        this.getLiftedState(store, instance.filters),
+        instance
+      );
+    } catch (e) {
+      this.relay('ERROR', e.message, instance);
+    }
+  };
+
   handleMessages = (message) => {
     if (
       message.type === 'IMPORT' || message.type === 'SYNC' && this.socket.id && message.id !== this.socket.id
     ) {
-      this.store.liftedStore.dispatch({
-        type: 'IMPORT_STATE', nextLiftedState: parse(message.state)
-      });
+      this.importPayloadFrom(this.store, message.state, this.instance);
```

Comment:
1. What's the point of supporting a custom `store` parameter in `getLiftedState` and `getLiftedStateRaw` if it's just set to `this.store` anyways?
2. What's the difference between `this.filters` and `this.instance.filters`? I can't find where `this.instance.filters` is set...